requirements/default.in

```diff
@@ -7,7 +7,7 @@ bleach~=3.2.1
 boto3>=1.2.3  # pip-compile does not handle extras properly (django-storages)
 celery==4.4.3
 commonware~=0.6.0
-cryptography~=2.9.2  # pip-compile does not handle extras properly (requests)
+cryptography~=3.2.1  # pip-compile does not handle extras properly (requests)
```

Comment:
Major number update, with [some backwards incompatible changes](https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst#321---2020-10-27), but we don't use this module directly, and the other modules which depend on it are happy with the version update. Doing some limited tested locally everything seems to work.